.circleci/config.yml

```diff
@@ -0,0 +1,210 @@
+version: 2.1
+
+jobs:
+  build:
+    docker:
+      - image: cloudreach/sceptre-circleci:0.8.0
+
+    steps:
+      - checkout
+
+      - restore_cache:
+          key:
+            v1-dependency-cache-{{ checksum "setup.py" }}-{{ checksum "Makefile"
+            }}
```

Comment:
Why are these curly braces on the next line? Seems like they should just be moved back up

Reply:
fixed